.husky/pre-commit

```
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -eo pipefail
+# set -eo pipefail
```

I'm using Ubuntu 22.10, I have no idea why this makes `pnpm cz` fail:
```
? Does this change affect any open issues? No
.husky/pre-commit: 2: set: Illegal option -o pipefail
git exited with error code 1
```

AFAIK `-o pipefail` is not an illegal option?

---

It’s because in ubtunu `sh` is symlinked to `dash` which doesn’t support this option. Fix should be to replace the shebang on the first line with `#!/bin/bash`

---

This didn't work for me, I got the  same error. I also tried `#!/usr/bin/bash` and other variants.

This works:
```
#!/bin/bash
set -e pipefail
```
According to a quick google this still fails early in pipelines, but I'm not sure. I can try and open up a new PR instead on my Mac, where this commit should just work (tm).